scripts/setup-obproxy.sh

```diff
@@ -84,6 +85,11 @@ while [[ $# -gt 0 ]]; do
       shift
       ;;
     -l|--list)
+      echo -e "\nOBProxy Deployments: \n"
+      kubectl get deployment -n $NAMESPACE -l obproxy.oceanbase.com/obpxory -L obproxy.oceanbase.com/for-obcluster -o wide
+      exit 0
+      ;;
+    -A|--list-all)
```

Comment:
-A only means all namesapces, still need -l to list
and the complete form should be --all

Reply:
Yes, `-l|--list` are still there and are scoped with `$NAMESPACE`. The diff does not show the line upon line 89.